src/hooks/v2/contractReader/ProjectSplits.ts

```diff
@@ -20,5 +45,11 @@ export default function useProjectSplits({
       projectId && domain
         ? [projectId.toHexString(), domain, splitGroup]
         : null,
+    formatter: useCallback((value): Split[] => {
```

Comment:
The contract read returns BigNumbers, but we can't store that in Redux, and in general its less convenient. The maximum values for all of these properties _doesn't_ exceed the JS max value, so we're safe.

https://github.com/jbx-protocol/juice-contracts-v2/blob/3671aa3f10de661dad9e99db1ecc574a6f4a3b89/contracts/libraries/JBConstants.sol